src/Datadog.Trace.ClrProfiler.Managed/ScopeFactory.cs

```diff
@@ -62,5 +64,81 @@ public static Scope CreateOutboundHttpScope(Tracer tracer, string httpMethod, Ur
             // or we couldn't populate it completely (some tags is better than no tags)
             return scope;
         }
+
+        public static Scope CreateDbCommandScope(Tracer tracer, DbCommand command, string integrationName)
+        {
+            if (!tracer.Settings.IsIntegrationEnabled(integrationName))
```

Comment:
How do I know that Settings is not null? The Tracer default constructor sets it to null. <noob-question/>

Reply:
Thanks @zacharycmontoya for pointing out the null coalescing https://github.com/DataDog/dd-trace-dotnet/blob/master/src/Datadog.Trace/Tracer.cs#L55

Reply:
Good question. The default ctor calls the internal ctor with all possible parameters. That one sets `Tracer.Settings` to a default value if the argument it null.

https://github.com/DataDog/dd-trace-dotnet/blob/5b101d00d46e523f9ace1fe6a4f954267d65466a/src/Datadog.Trace/Tracer.cs#L52-L55